three-node-cluster/3-node-cluster.yml

```diff
@@ -0,0 +1,659 @@
+      FileSystemId: !Ref WSO2ISEFSFileSystemsuffix
+      SecurityGroups:
+        - !Ref WSO2ISEFSSecurityGroupsuffix
+  WSO2ISPrivateSubnet2EFSMountTargetsuffix:
+    Type: 'AWS::EFS::MountTarget'
+    Properties:
+      SubnetId: !Ref WSO2ISPrivateSubnet2suffix
+      FileSystemId: !Ref WSO2ISEFSFileSystemsuffix
+      SecurityGroups:
+        - !Ref WSO2ISEFSSecurityGroupsuffix
+  WSO2ISPrivateSubnet3EFSMountTargetsuffix:
+    Type: 'AWS::EFS::MountTarget'
+    Properties:
+      SubnetId: !Ref WSO2ISPrivateSubnet3suffix
+      FileSystemId: !Ref WSO2ISEFSFileSystemsuffix
+      SecurityGroups:
+        - !Ref WSO2ISEFSSecurityGroupsuffix
+  # Product configurations
+  BastionSecurityGroupsuffix:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      VpcId: !Ref WSO2ISVPCsuffix
+      GroupDescription: WSO2 Bastion node Security Group for 5.3
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+          CidrIp: 0.0.0.0/0
+  WSO2ISSecurityGroupsuffix:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      VpcId: !Ref WSO2ISVPCsuffix
+      GroupDescription: WSO2 Identity Manager Security Group
+      GroupName: WSO2SecurityGroup
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: '8280'
+          ToPort: '8280'
+          SourceSecurityGroupId: !Ref BastionSecurityGroupsuffix
+        - IpProtocol: tcp
+          FromPort: '9763'
+          ToPort: '9763'
+          SourceSecurityGroupId: !Ref BastionSecurityGroupsuffix
+        - IpProtocol: tcp
+          FromPort: '9443'
+          ToPort: '9443'
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: '8243'
+          ToPort: '8243'
+          SourceSecurityGroupId: !Ref BastionSecurityGroupsuffix
+        - IpProtocol: tcp
+          FromPort: '5672'
+          ToPort: '5672'
+          SourceSecurityGroupId: !Ref BastionSecurityGroupsuffix
+        - IpProtocol: tcp
+          FromPort: '5701'
+          ToPort: '5701'
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: '8140'
+          ToPort: '8140'
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: '4000'
+          ToPort: '4000'
+          CidrIp: 0.0.0.0/0
+  WSO2BastionInstancesuffix:
+    Type: 'AWS::EC2::Instance'
+    Properties:
+      DisableApiTermination: 'false'
+      InstanceInitiatedShutdownBehavior: stop
+      ImageId: !FindInMap
+        - WSO2ISAMIRegionMapsuffix
+        - !Ref 'AWS::Region'
+        - AMI
+      InstanceType: !Ref BastionInstanceType
+      KeyName: !Ref KeyPairName
+      Monitoring: 'false'
+      Tags:
+        - Key: Name
+          Value: suffix_WSO2BastionInstance
+        - Key: User
+          Value: 'sachinm@wso2.com'
```

Comment:
Should be provided at the runtime

Reply:
Removed by https://github.com/wso2/performance-is/pull/128/commits/ef88a709e62a267d93e2cf768f66851ed0a8035e